pyproject.toml

```diff
@@ -9,3 +9,7 @@ build-backend = "setuptools.build_meta"
 testpaths = [
     "tests",
 ]
+
+[isort]
```

Comment:
You could consider adding `force_single_line = true` here. As far as I am concerned I think it looks better and makes imports easier to read. But as you prefer.

Reply:
Sure, added it.